packages/core/contracts/snapshot-helpers/SnapshotVotingPower.sol

```diff
@@ -0,0 +1,58 @@
+// SPDX-License-Identifier: AGPL-3.0-only
+pragma solidity 0.8.16;
+
+/**
+ * @title VotingInterface
+ * @notice Minimum interface required to interact with VotingV2 contract.
+ */
+interface VotingInterface {
+    struct VoterStake {
+        uint128 stake;
+        uint128 pendingUnstake;
+        uint128 rewardsPaidPerToken;
+        uint128 outstandingRewards;
+        int128 unappliedSlash;
+        uint64 nextIndexToProcess;
+        uint64 unstakeTime;
+        address delegate;
+    }
```

Comment:
Is this used?

Reply:
you're correct. fixed in 6618296b3d3882e2ded245c27d6fa1b158b8f06a